docs/design/scheduler_object_states.md

```diff
@@ -67,6 +69,7 @@ The events that can trigger a state change:
 * Wait: mark an application as idle (source: core scheduler)
 * Complete: mark an application as complete (source: resource manager)
 * Kill: kill an application (source: resource manager)
+* Expire: progress the application to the expired state and remove it from the scheduler (source: core scheduler)
 
 Here is a diagram that shows the states with the event that causes the state to change:  
 ![application state diagram](./../assets/application-state.png)
```

Comment:
We need to update this diagram. Let me know if you need help on generating the diagram.

Reply:
The looplab/fsm code allows generating a graphviz file via `fsm.Visualize(state-machine)`.
This file can be converted using the `dot` tool or other online tools into a svg or png file.

Just tracking for posterity.